README.md

```diff
@@ -58,9 +58,16 @@ keyring-util function userid keyring label
   * `EXPORT` - exports a certificate in PEM format. The file is created in a `pwd` directory with a name of `<cert_alias>.pem`
     - Supported Arguments:
         * `-l <label>`: Required. Specifies the certificate to be exported by label.
+        * `-f </path/to/output>`: Required. Specifies where to write out the exported certificate.
```

Comment:
Not related to this PR, but it looks like lines 35-37 should be `LISTRING` instead of `NEWRING`.